sawmill-core/src/test/java/io/logz/sawmill/utils/FactoryUtils.java

```diff
@@ -14,13 +16,16 @@
 import java.util.Map;
 
 public class FactoryUtils {
+
     public static final TemplateService templateService = new TemplateService();
-    public static final ConditionParser conditionParser;
+    public static final ProcessorFactoryRegistry defaultProcessorFactoryRegistry = new ProcessorFactoryRegistry(
+            new ProcessorFactoriesLoader(templateService, new GeoIpConfiguration("GeoIP2-City-Test.mmdb"))
```

Comment:
It is strange to see here the specific geoip database configuration. I think it is more clear that the default will be without the geoip configuration. 
Let's be more explicit and use the geoip configuration only on `GeoIpProcessor`.
To achieve this, you can create a new `createProcessor` method that accepts also `SawmillConfiguration...` and will create a specific `ProcessorFactoryRegistry` with the given configuration